### PR TITLE
MUSARK-753 - Refactoring the whole picklist + adding support for customizing template

### PR DESCRIPTION
**Pull request by jarlah**
Adding support for customizing template #4 and #5

### CODE REVIEW

**Review by steinaol — APPROVED** [review]
LGTM

File: src/modules/picklist/PickListContainer.js [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+  loadChildren$,
+  loadNode$
+};
+
+const customProps = {
+  updateMoveDialog,
+  emitError,
+  emitSuccess,
+  showModal,
+  moveNode: MusitNode.moveNode(),
```
Comment (jarlah) [inline src/modules/picklist/PickListContainer.js:196]: Ajaj must remove this

File: src/modules/picklist/PickListContainer.js [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+  moveObject: MusitObject.moveObject(),
```
Comment (jarlah) [inline src/modules/picklist/PickListContainer.js:197]: Ajaj must remove this

File: src/modules/picklist/PickListContainer.js [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+  isTypeNode: (props) => 'nodes' === props.route.type,
```
Comment (jarlah) [inline src/modules/picklist/PickListContainer.js:198]: Should Move this back down to component.

File: src/modules/print/PrintTemplateComponent.js [hunk 4]
```diff
@@ -64,6 +83,18 @@ export class ChooseTemplate extends Component {
...
         {' '}
+        {this.canSelectPath(this.props.store.templateId) &&
+        <select
+          className="printTool level"
+          onChange={this.selectLevel}
+          defaultValue={this.props.store.level}
+        >
+          <option value={-2}>To nærmeste</option>
+          <option value={2}>Fra bygg</option>
+          <option value={0}>Hele stien</option>
```
Comment (jarlah) [inline src/modules/print/PrintTemplateComponent.js:94]: Should remove "Hele stien" and default to "Fra bygg"

Comment (jarlah) [reply src/modules/print/PrintTemplateComponent.js:94]: Vups. Should translate all options

**Review by jarlah — COMMENTED** [review]
Need to take a look at this again

**Pull request merged**